specifications/cdsc-wg1-02.md

```diff
@@ -1 +1,799 @@
+* `cds_grants_api` - _URL_ - (REQUIRED) The base url for the [Grants API](#grants-api)
+* `cds_directory_api` - _URL_ - (REQUIRED) The base url for the [Directory API](#directory-api)
+* `cds_scope_descriptions` - _Map[[ScopeDescription](#scope-descriptions-format)]_ - (REQUIRED) An object providing additional information about what Scope values listed in `scopes_supported` mean. This object MUST include a key for each Scope listed in `scopes_supported` with a [Scope Description](#scope-descriptions-format) as that key's value.
+* `cds_registration_fields` - _Map[[RegistrationField](#registration-field-format)]_ - (REQUIRED) An object providing additional information about Registration Field that are referenced in [Scope Description's](#scope-descriptions-format) `registration_requirements` list. This object MUST include a key for each Registration Field `id` included in the `registration_requirements` lists in the metadata object with a [Registration Field](#registration-field-format) as that key's value. If all `registration_requirements` lists are empty, this reference object is an empty object (`{}`).
+
+Other values not mentioned here but listed in specifications for the Authorization Server Metadata specification or other extensions MAY be included as described in their respective specifications.
+
+### 3.3. Scope Descriptions Object Format <a id="scope-descriptions-format" href="#scope-descriptions-format" class="permalink">🔗</a>
+
+Scope Description objects are formatted as JSON objects and contain named values. The following values are included in the default list available in scope description objects.
+
+* `id` - _string_ - (REQUIRED) The unique identifier of the scope. This MUST be the same value as the object key the Metadata Object's `cds_scope_descriptions` object.
+* `name` - _string_ - (REQUIRED) A human-readable name of the scope.
+* `description` - _string_ - (REQUIRED) A human-readable description of what access or actions the scope will enable.
+* `documentation` - _URL_ - (REQUIRED) Where developers can find documentation for the scope.
+* `registration_requirements` - _Array[string]_ - (REQUIRED) A list of any registration fields that MUST be submitted with [Client Registration Requests](#registration-request) or requirements that must be completed after registration before the scope is available to use in the Server's production environment. All values in this array MUST be included in the Metadata Object's `cds_registration_fields` object for a complete description of the requirement. If no additional requirements are needed, this value is an empty array (`[]`).
+* `registration_optional` - _Array[string]_ - (REQUIRED) A list of any registration fields that Clients MAY submit with the [Client Registration Requests](#registration-request). If no additional fields are available, this value is an empty array (`[]`).
```

Comment:
From @antarasargam ([on April 24, 2024](https://github.com/carbon-data-specification/Customer-Data/pull/25#discussion_r1578662918)):

---

nit: But why is this a REQUIRED field? 

---